pkgs/applications/kde/rocs.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv, mkDerivation, makeWrapper, lib,
+  extra-cmake-modules, boost,
+  qtbase, qtscript, qtquickcontrols, qtwebkit, qtxmlpatterns, grantlee,
+  kdoctools, karchive, kxmlgui, kcrash, kdeclarative, ktexteditor, kguiaddons
+}:
+
+mkDerivation {
```

Comment:
Importing mkDerivation instead of using `stdenv.mkDerivation` is pretty uncommon in nixpkgs. Is there a reason for doing it this way?

Reply:
No reason - I based this derivation on another KDE application (I think it was Okular) where it was done this way.

Reply:
It's a custom mkDerivation function to build KDE applications (defined in `pkgs/applications/kde/default.nix`).